### PR TITLE
[DCFS-27] Develop an E-commerce Store Database

### PR DESCRIPTION
**Pull request by kemysola**
## Task: `Databases`
- `Changes you made to given task`
- Good morning @adaugochi @Ireti203 .Kindly review my pull request.Thank you.

### CODE REVIEW

File: submissions/adekemiAwomolo/database/init.sql [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+VALUES('Aneke Paul', 'neke@hotmail.com','america' ,'234typo', '0706099884', now()),
+('Paul Panda', 'pai23@hotmail.com','canada', 'paulpp', '0706099894', now()),
+('Kmei Sanders', 'pai23@hotmail.com','canada', 'lekmpp', '0706033384', now());
+
+SELECT * FROM admins;
+
+/*product_details database*/
+CREATE TABLE products_details(
+id INT NOT NULL AUTO_INCREMENT,
+productCategory_id INT NOT NULL,
```
Comment (adaugochi) [inline submissions/adekemiAwomolo/database/init.sql:44]: There is category here

File: submissions/adekemiAwomolo/database/init.sql [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+product_name VARCHAR(100) NOT NULL,
+description TEXT NOT NULL,
+images BLOB NOT NULL,
+admin_id INT NOT NULL,
+unit_price DECIMAL (9,2) NOT NULL,
+stock_quantity INT NOT NULL,
+stock_status VARCHAR(200) NOT NULL,
+category_id INT NOT NULL,
```
Comment (adaugochi) [inline submissions/adekemiAwomolo/database/init.sql:52]: and here too

File: submissions/adekemiAwomolo/database/init.sql [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+/*single_order database*/
+
+CREATE TABLE single_orders(
+orderItemId int not null auto_increment,
+ OrderId int not null,
+ productId int not null,
+ quantity int not null,
+ unitPrice decimal not null,
+ totalAmount decimal(10, 2) not null,
```
Comment (adaugochi) [inline submissions/adekemiAwomolo/database/init.sql:134]: Kindly follow a single pattern. Is it `snake_case` or `camelCase` or `PascalCase`. Follow a single convention

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/adekemiAwomolo/database/init.sql (hunk 1 above)
Comment (kemysola) [reply submissions/adekemiAwomolo/database/init.sql:44]: Good afternoon @adaugochi . I do not understand ma. am I to put in a category table here or to remove it ma

**Comment by kemysola** [comment]
Good afternoon @adaugochi  Category is included in the tables you highlighted and i have corrected the casings pattern to snake-casing.Kindly review.Thank you

File: submissions/adekemiAwomolo/database/init.sql (hunk 3 above)
Comment (kemysola) [reply submissions/adekemiAwomolo/database/init.sql]: corrected it ma

File: submissions/adekemiAwomolo/database/init.sql [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+(3, 3, 2500, now());
+
+SELECT * FROM orders;
+
+
+/*single_order database*/
+
+CREATE TABLE single_orders(
+order_item_id int not null auto_increment,
+ Order_id int not null,
```
Comment (adaugochi) [inline submissions/adekemiAwomolo/database/init.sql:130]: first letter should be in lowercase

File: submissions/adekemiAwomolo/database/init.sql [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+(2, 2, 5000, now()),
+(3, 3, 2500, now());
+
+SELECT * FROM orders;
+
+
+/*single_order database*/
+
+CREATE TABLE single_orders(
+order_item_id int not null auto_increment,
```
Comment (adaugochi) [inline submissions/adekemiAwomolo/database/init.sql:129]: follow one single pattern. you have always used `id ` to indicate primary key

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by adaugochi — APPROVED** [review]

**Pull request merged**